packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx

```diff
@@ -151,7 +152,15 @@ export const HvCanvasSidePanel = forwardRef<
         aria-label={open ? labels.close : labels.open}
       >
         <div className={classes.handleButton}>
-          {open ? <Start color={["primary"]} /> : <End color={["primary"]} />}
+          {open ? (
+            <HvTooltip title={labels.close}>
+              <Start color={["primary"]} />
+            </HvTooltip>
+          ) : (
+            <HvTooltip title={labels.open}>
+              <End color={["primary"]} />
+            </HvTooltip>
+          )}
```

Comment:
We should be able to replace all of this with:

```tsx
<HvIconButton
  variant="primaryGhost"
  title={open ? labels.close : labels.open}
  onClick={handleTogglePanel}
  className={classes.handle}
>
  <End style={{ rotate: open ? "180deg" : undefined }} />
</HvIconButton>
```

and remove many of the styles too

Reply:
Yeah, I updated I removed this clutter.

Reply:
there's still quite a few redundant/duplicated styles BTW

Reply:
Yeah, I removed the redundant styles.